### PR TITLE
add: Writing style guide

### PR DESCRIPTION
**Pull request by ghost**
# Description
Creation of a Writing Style Guide as reference for all Thoth Tech communication (documentation and website)
More in-depth understanding of writing style built on existing documentation workshop and research.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from Area Leads and Docs Team on the Pull Request

### CODE REVIEW

**Comment by ghost** [comment]
Review requested, but still intentionally a draft PR. Looking for feedback to action as I go 😄

File: docs/documentation/writing-style-guide.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+**To Educate:** We want to assist our people in their learning and provide them with the information they need
+
+**To Guide:** We want to lead our people in a thoughtful manner towards their goals
+
+**To be Honest:** We want to be truthful to our people and focus on our real strengths
+
+While writing, it is important to:
+
+1. Know your audience
+2. Know you intention of the writing
```
Comment (kats8) [inline docs/documentation/writing-style-guide.md]: Typo :) - (Know the intention of your writing?)

Comment (ghost) [reply docs/documentation/writing-style-guide.md]:
```suggestion
2. Know the intention of your writing
```

**Comment by ghost** [comment]
I did not mean to add these commits into this branch😢 
- fix: remove teams 07dc2fccc90f1865fb4fe5cd0913267362adea35
- fix: update readme and add week6 announcement 20134b64d427b46c9177a78a15689bef368b086d

Have added a new branch as I originally intended

**Review by j-6 — COMMENTED** [review]
@talzid a few edits made. Looks really good.

**Review by ginge270 — APPROVED** [review]

**Comment by ghost** [comment]
Added information based on Vale suggestions (#86). The two files should now have the same message and suggestions - along with the comments proposed in the review.

**Comment by ghost** [comment]
Added information based on Vale suggestions (#86). The two files should now have the same message and suggestions - along with the comments proposed in the review.

**Review by tancnle — APPROVED** [review]
Thank you for your hard work on this @talzid 🙇🏼 Looks bonza to me. 🚀 👍🏼

**Pull request merged**